README.md

```diff
@@ -21,6 +21,9 @@ For Resources in Polish, visit: :poland: [Resources in Polish](Readme_PL.md)
 ## Scaling
 - :closed_book: [Craig Larman, Bas Voddle - Large-Scale SCrum: More with LeSS](https://www.goodreads.com/book/show/31378603-large-scale-scrum)
 
+## Scrum Masters Development and Growth Paths
```

Comment:
Why such name? Doesn't "Scrum Master" enough?

Reply:
What do you think about "Scrum Master Development"? I wanted to highlight that this section is about how to develop SM skills, measure efficiency, etc. It's a complex topic and may be interesting especially to those that work alone or are responsible for evaluating and developing SMs in their organizations.